docs/spec/staking/state.md

```diff
@@ -2,46 +2,46 @@
 
 ### Pool
 
-The pool is a space for all dynamic global state of the Cosmos Hub.  It tracks
-information about the total amounts of Atoms in all states, moving Atom
-inflation information, etc.
+The pool dynamically tracks information about the total amounts of staking
+tokens and their states (bonded or loose). Note that `NotBondedTokens` _does_
+include both `unbonding` tokens, and fully `unbonded` tokens. 
 
  - Pool: `0x01 -> amino(pool)`
 
 ```golang
 type Pool struct {
-    LooseTokens         sdk.Int   // tokens not associated with any bonded validator
-    BondedTokens        sdk.Int   // reserve of bonded tokens
+    NotBondedTokens sdk.Int   // tokens not associated with any bonded validator
+    BondedTokens    sdk.Int   // reserve of bonded tokens
 }
 ```
 
 ### Params
 
 Params is global data structure that stores system parameters and defines
-overall functioning of the staking module. 
+overall functioning of the staking module. it is stored 
 
- - Params: `0x00 -> amino(params)`
+ - Params: `Paramsspace("staking") -> amino(params)`
 
 ```golang
 type Params struct {
-    MaxValidators uint16 // maximum number of validators
-    BondDenom     string // bondable coin denomination
+	UnbondingTime time.Duration // time duration of unbonding
+    MaxValidators uint16        // maximum number of validators
+    BondDenom     string        // bondable coin denomination
 }
 ```
 
 ### Validator
 
 Validators are identified according to the `OperatorAddr`, an SDK validator
```

Comment:
The next two paragraphs contain information about the different validator keys without explaining the difference between the keys (how to differentiate them visually, or in code) or what the functions of each are. I think we should add some more explanation here.

Reply:
This spec is not supposed to specify detailed function names/specifics of implementation, only the key-value specifics (as is done). It is up to the implementer of the spec to decide what the names of these functions should be called, hence function names have not been included. 

However, I've updated (and combined) this top section to explain to summarize the uses of each key/index. Further I've also improved the paragraphs following the key-bulleted list to better describe the more detailed usage.